bin/ycsb

```diff
@@ -76,6 +76,7 @@ DATABASES = {
     "geode"        : "com.yahoo.ycsb.db.GeodeClient",
     "googlebigtable"  : "com.yahoo.ycsb.db.GoogleBigtableClient",
     "googledatastore" : "com.yahoo.ycsb.db.GoogleDatastoreClient",
+    "griddb"       : "com.yahoo.ycsb.db.GridDBClient",
```

Comment:
this should point to `com.yahoo.ycsb.db.griddb.GridDBClient` now

Reply:
I changed package name for GridDB by 81cf64a83c75ebc6503500ab3810be6a21d2e899 .